nvtabular/__init__.py

```diff
@@ -25,11 +25,12 @@
 
 
 ColumnGroup = column_group.ColumnGroup
+ColumnSelector = column_selector.ColumnSelector
```

Comment:
I'm good with an alias of some kind to make the client code a bit terser. I'm not sure about `Select`, because everything that follows after a `>>` operator is framed as noun rather than a verb. I still kinda lean toward `Cols` because it makes clear what is being operated on, but open to options here.

Reply:
This is a good point.  I guess my discomfort/confusion was in the subtle differences between the old `ColumnGroup` name and the new `ColumnSelector` name.  I guess the `ColumnSelector` name implies that the selection has not happened yet, whle `ColumnGroup` implies that a selection was already made.  Therefore, when a `ColumnSelector` is initialized with a literal  list of column names, the `ColumnSelector` name is a bit confusing.

In my mind,  I was imagining something like `select` as a helper/fatory function that would produce a `ColumnSelector`, but that would probably make more sense if `ColumnSelector` was called `ColumnSelection` (or something).

Reply:
Yeah, I see the confusion you're highlighting. The thing is that the old language was a bit of a fib; `ColumnGroup` wasn't a group of columns, it was only a potential group of columns since no selection had occurred yet. The selection of literal columns of data from a dataset (forming an actual group of columns) was happening inside the `Operator.fit/transform` methods, which took place long after the point in graph construction where `ColumnGroup` was instantiated. It looked simple, but it was giving people (e.g. me) a subtly incorrect mental model for understanding what was happening.

(I found this really confusing when I started using NVTabular. Why doesn't `ColumnGroup` have any apparent relationship to a dataset? Where does that link happen? If I'm handing it column names, why isn't it really doing anything with them?)

Reply:
Now that the `["a", "b", "c"] >> ops.Operator()` syntax has been restored, I'm not sure we need to worry too much about providing a shorter alias for `ColumnSelector` anymore (or at least I think we can postpone that decision until we actually want to deprecate the list rshift syntax.)